backend/tests/conftest.py

```
@@ -0,0 +1,605 @@
+            return MockResponse(401, {"detail": "Invalid authorization token"})
+        
+        # Success response for file upload
+        if url == "/speech_profile/v3/upload-audio" and files:
+            return MockResponse(200, {"url": "https://storage.googleapis.com/test-bucket/test.wav"})
+        
+        return MockResponse(200, {"data": []})
+
+mock_testclient = MagicMock()
+mock_testclient.TestClient = MockTestClient
+
+def mock_get_current_user_uid(authorization: str = None):
```

Ruff

Redefinition of unused `mock_get_current_user_uid` from line 118

(F811)